File: emarket-ui/src/models/zaloPayOrder.js
export default class ZaloPayOrder {
    constructor(
        app_id = 2554,
        app_user,
        app_time,
        app_trans_id,
        amount,
        bank_code = "",
        embed_data = JSON.stringify({redirecturl: "http://localhost:8080/myPurchase"}),
        item = JSON.stringify([]),
        callback_url="http://localhost:8085/api/zaloPay/callback",
        description,
        mac,
        key1 = "sdngKKJmqEMzvh5QQcdD2A9XBSKUNaYn",
        key2 = "trMrHtvjo6myautxDUiAcYsVtaeQ8nhf",
    ) {
        Object.assign(this, {
            app_id,
            app_user,
            app_time,
            app_trans_id,
            amount,
            bank_code,
            embed_data,
            item,
            callback_url,
            description,
            mac,
            key1,
            key2,
        });
    }
}

File: EMarket/src/main/java/DoAn/B19DCCN445/EMarket/dto/ZaloPayOrder.java
package DoAn.B19DCCN445.EMarket.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Builder
public class ZaloPayOrder {
	private int app_id;
	private String app_user;
	private Long app_time;
	private String app_trans_id;
	private Double amount;
	private String bank_code;
	private String embed_data;
	private String item;
	private String callback_url;
	private String description;
	private String mac;

}
